### PR TITLE
Upgrade mathjax and adjust styling to fix rendering.

### PR DESCRIPTION
**Pull request by gunnarvelle**
Setter nyeste mathjax. Måtte endre litt på stylinga for at ikkje formler skulle kuttes, pga endring i bredde på feks vektorer.

Gammel mathjax:
image

Ny mathjax:
image

Vektorpiler er penere opptegna. Avstand mellom formler er litt større på grunn av padding, men det måtte til for ikkje å kutte noen av vektorene:
image
image

### CODE REVIEW

**Comment by haattis** [comment]
Er flere formler som har blitt presset tettere sammen. Under er to eksempler.

Utsnitt fra artikkel 31308 i PR-instans:
image
image

**Comment by gunnarvelle** [comment]
Dette har faktisk med måten formelen er lagra. 'og' i begge eksempla er en del av matteformelen og er satt inn uten ekstra mellomrom.
```
<math xmlns="http://www.w3.org/1998/Math/MathML" xlmns="xmlns=&quot;http://www.w3.org/1998/Math/MathML">
  <math xmlns="http://www.w3.org/1998/Math/MathML">
    <math xmlns="http://www.w3.org/1998/Math/MathML">
      <mover>
        <mi>u</mi>
        <mo>&#x2192;</mo>
      </mover>
      <mi>og</mi>
      <mover>
        <mi>v</mi>
        <mo>&#x2192;</mo>
      </mover>
    </math>
  </math>
</math>
```

**Comment by gunnarvelle** [comment]
Mellomrom blei bedre når eg endra tilbake til mathmlSpacing: false

**Comment by haattis** [comment]
Synes fortsatt blokker med pipes/parantes er for tette uten padding. Åpner man samme formel i mathml-editoren vises det padding.

Denne formelen...:
image

... er fortsatt ganske tett på "og" sammenliknet med i master:
image
